### PR TITLE
refac: CardView

### PR DESCRIPTION
**Pull request by yuritoledo**
Refactor CardView, to functional component.

Fixed eslint in some files.

### CODE REVIEW

File: src/CardView/styles.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  cardContainer: {},
+  cardFace: {},
+  icon: {
+    position: 'absolute',
+    top: 15,
+    right: 15,
+    width: 60,
+    height: 40,
+  },
+  resizeMode: 'contain',
```
Comment (siliconavengers) [inline src/CardView/styles.js:16]: I think the `resizeMode` must be inside the icon obj.

File: src/CardView/CardView.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  Image,
+  Text,
+  Platform,
+} from 'react-native'
+
+import defaultIcons from '../Icons'
+import FlipCard from 'react-native-flip-card'
+import styles from './styles'
+
+const BASE_SIZE = { width: 300, height: 190 }
```
Comment (wagnerdelima) [inline src/CardView/CardView.js:15]: @yuritoledo I didn't manage to see the code running. Don't you think it's a good idea to set the width and height of the screen? We can do that using the Dimensions from react native.

https://facebook.github.io/react-native/docs/dimensions

**Review by wagnerdelima — APPROVED** [review]

**Comment by wagnerdelima** [comment]
@yuritoledo I saw you translated components labels and placeholders to Portuguese. I think we should keep them in English and then add internationalisation to these components.

File: src/CardView/CardView.js (hunk 2 above)
Comment (yuritoledo) [reply src/CardView/CardView.js:15]: I agree with you, but in this PR, I just refactor it to functional, I didn't focus on business logic.

We can add some `// TODO` into pieces of code like this to know where we can improve.

File: src/CardView/styles.js (hunk 1 above)
Comment (yuritoledo) [reply src/CardView/styles.js:16]: I didn't worked on style refac yet, this one is legacy.

**Comment by yuritoledo** [comment]
@wagnerdelima about the translate, I added it in portuguese, because I'm using this repo in a project. I will back it to english and then make a internationalization flag.

**Comment by wagnerdelima** [comment]
@yuritoledo this is good news. Thanks for the update.

**Comment by wagnerdelima** [comment]
I am sending you an email with some question if you don't mind.

**Comment by yuritoledo** [comment]
@wagnerdelima today I'll update the placeholders in another branch, because it's another feature/bug :grin: 

I already answered your email :)

**Pull request merged**